### PR TITLE
Fix cursor on user settings table

### PR DESCRIPTION
**Pull request by danxuliu**
The table shown in the user settings used a pointer cursor when hovering on _Full name_, _Password_ and _Quota_ cells to communicate the user that the cells could be clicked.

This was wrong for the _Quota_ cells, as in this case clicking on the cell itself made nothing; you had to click on the select element itself for the dropdown to be shown.

Besides that, the email address cells (hidden by default; you have to show them through _Show email address_ in the _Settings_ in the app navigation bar) showed the default cursor, while it should show the pointer cursor instead (as, like with _Full name_ and _Password_, clicking anywhere on the cell showed the input field).

This pull request fixes those issues and also removes some unneeded CSS rules related to the quota cell.

### CODE REVIEW

**Comment by rullzer** [comment]
@nextcloud/designers please have a look

**Review by skjnldsv — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Somehow this PR breaks the create user input fields:

master:

bildschirmfoto 2017-09-27 um 15 43 38


this branch:

bildschirmfoto 2017-09-27 um 15 43 53

**Comment by skjnldsv** [comment]
What? Strange, I did not had this issue. Let me try again! :)

**Comment by MorrisJobke** [comment]
> What? Strange, I did not had this issue. Let me try again! :)

I also thought it was a caching issue, but it actually was there after clearing the caches and not there on master.

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Let me try again

**Review by skjnldsv — APPROVED** [review]
Nope! Works fine here! :)
capture d ecran_2017-09-27_16-55-05

**Review by MorrisJobke — APPROVED** [review]
Re-tested and works now 👍
